### DISCUSSION TITLE
Exception in thread "main" java.io.IOException: Cannot run program "scancode.bat": CreateProcess error=2, 系统找不到指定的文件。

### DISCUSSION THREAD

**Discussion by ljhlovw** [oss-review-toolkit/ort#9559] [Q&A]
I'm using the Windows version, and when I scan the file, I get an error that the scancode.bat file can't be found. What am I going to do?
this is my commond ort.bat --debug scan -i D:/tools/tools/analyzer/analyzer-result.json -f JSON -o E:\tools\tools\scanners。
![image](https://github.com/user-attachments/assets/046cd835-718f-46fc-a161-2e73f53b6113)

**Comment by sschuberth** [comment]
If you're not using the Docker image (which comes with all requirements pre-installed), you need to manually ensure that all tools listed by `ort requirements` are installed on your system and part of your `PATH` environment. Also see e.g. [this](https://github.com/oss-review-toolkit/ort/issues/7893#issuecomment-1816705111) answer.
